ejb3/src/main/java/org/jboss/as/ejb3/remote/AssociationImpl.java

```diff
@@ -217,6 +230,34 @@ public CancelHandle receiveInvocationRequest(@NotNull final InvocationRequest in
         return cancellationFlag::cancel;
     }
 
+    private static <A> boolean intersect(final Set<A> fst, final Set<A> snd) {
+        fst.retainAll(snd);
+        return !fst.isEmpty();
+    }
+
+    private static Set<String> typesOf(final Object o) {
```

Comment:
This shouldn't come up for the TCK since the spec has no concept of client-side interception.

Just wanted to conclude that if the purpose of the big exclusion thing is to just remove the weld stuff, then let's look at removing that stuff in the interceptors instead.